doc/stdlib/io-network.rst

```diff
@@ -168,11 +168,11 @@ General I/O
 
    See ``read`` for a description of the ``all`` option.
 
-.. function:: read(stream::IO, nb=typemax(Int); all=true)
+.. function:: read(s::IOStream, nb::Integer; all=true)
```

Comment:
It doesn't have the `all` kwarg though. You can't do `read(s; all=false)`, can you?